draft-ietf-quic-transport.md

```diff
@@ -395,17 +395,17 @@ Connection ID:
 : Octets 1 through 8 contain the connection ID. {{connection-id}} describes the
   use of this field in more detail.
 
-Packet Number:
-
-: Octets 9 to 12 contain the packet number.  {{packet-numbers}} describes the
-  use of packet numbers.
-
 Version:
 
 : Octets 13 to 16 contain the selected protocol version.  This field indicates
```

Comment:
You'll need to renumber the octets, too.

Reply:
Thanks Mike, fixed